Source/ios/Storyboard/UIView+TyphoonOutletTransfer.m

```diff
@@ -0,0 +1,198 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+//  TYPHOON FRAMEWORK
+//  Copyright 2015, Typhoon Framework Contributors
+//  All Rights Reserved.
+//
+//  NOTICE: The authors permit you to use, modify, and distribute this file
+//  in accordance with the terms of the license agreement accompanying it.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#import "UIView+TyphoonOutletTransfer.h"
+#import "NSLayoutConstraint+TyphoonOutletTransfer.h"
+#import <objc/runtime.h>
+
+@implementation UIView (TyphoonOutletTransfer)
+
+- (void)setTyphoonNeedTransferOutlets:(BOOL)typhoonNeedTransferOutlets {
+    objc_setAssociatedObject(self, @selector(typhoonNeedTransferOutlets), @(typhoonNeedTransferOutlets), OBJC_ASSOCIATION_RETAIN_NONATOMIC);
+}
+
+- (BOOL)typhoonNeedTransferOutlets {
+    return [objc_getAssociatedObject(self, @selector(typhoonNeedTransferOutlets)) boolValue];
+}
+
+
+// Swizzle didMoveToWindow
++ (void)load {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        Class class = [self class];
```

Comment:
Method swizzling in Typhoon is implemented by TyphoonMethodSwizzler

I think would be better use this util class instead your duplicate ;)